BSB_LAN/BSB_LAN.ino

```diff
@@ -4917,16 +4916,15 @@ int set(int line      // the ProgNr of the heater parameter
         break;
       }
       case VT_PPS_TIME:
-      {
-        int hour=0, minute=0, second=0;
-        strcpy_P(sscanf_buf, PSTR("%d.%d.%d"));
-        sscanf(val, sscanf_buf, &hour, &minute, &second);
-        setTime(hour, minute, second, weekday(), 1, 2018);
-//        printFmtToDebug(PSTR("Setting time to %d:%d:%d\r\n"), hour, minute, second);
-        pps_time_set = true;
-      }
-      case VT_HOUR_MINUTES:
-      {
+      case VT_HOUR_MINUTES: {
```

Comment:
No, these two are different cases, I just missed a `break` as you pointed out correctly.

Reply:
I leave comment with warnings and revert my changes.